src/shared/transactionDataHelper.ts

```diff
@@ -53,7 +42,7 @@ export const transformTransactionData = (tx: RawTransactionI): TransactionI => {
     transactionType,
     type: tx.type,
     hash: tx.hash,
-    amount: dfiAmount,
+    amount: formatEther(BigInt(tx.value ?? "0")),
```

Comment:
for diff coin decimal value will be different, I guess we should get decimal value and then format

Reply:
this one is only for "DFI value" so should always 18 decimals